l10n_it_fatturapa_export_zip/views/attachment_view.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="view_fatturapa_out_attachment_tree">
+        <field name="name">fatturapa.attachment.out.tree.exported</field>
+        <field name="model">fatturapa.attachment.out</field>
+        <field name="inherit_id" ref="l10n_it_fatturapa_out.view_fatturapa_out_attachment_tree"/>
+        <field name="arch" type="xml">
+            <field name="out_invoice_ids" position="after">
+                <field name="zip_exported"/>
+            </field>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_fatturapa_out_attachment_search">
+        <field name="name">fatturapa.attachment.out.search.exported</field>
+        <field name="model">fatturapa.attachment.out</field>
+        <field name="inherit_id" ref="l10n_it_fatturapa_out.view_fatturapa_out_attachment_search"/>
+        <field name="arch" type="xml">
+            <field name="name" position="after">
```

Comment:
Secondo me qua ci andrebbe un separatore, per mettere in AND 'ready' ed i filtri sullo ZIP:

![image](https://user-images.githubusercontent.com/1033131/52578933-7f8b1600-2e25-11e9-9485-e59517880667.png)

Reply:
fatto